src/lm_buddy/jobs/evaluation/hf_evaluate.py

```diff
@@ -99,14 +104,18 @@ def run_eval(config: HuggingFaceEvalJobConfig) -> Path:
     # Choose which model client to use
     if type(config.model) == VLLMCompletionsConfig:
         model_name = config.model.inference.base_url
+        output_model_name = config.model.inference.engine
+        if "mistral" in model_name:
+            # run the mistral client
+            logger.info(f"Using Mistral client. Endpoint: {model_name}")
+            model_client = MistralModelClient(model_name, config.model)
+        else:
```

Comment:
could there be a case that the model is not mistral and not openAI and we try to also run the OpenAI client on it? we might want an elif here if that's the case

Reply:
Not right now, i.e. all the other models (VLLM, llamafile) while not beign OAI use the OAI client (because they wanted to be compatible). But I agree we'll need to have elifs when we start using other clients (or, better, explicitly write a mapping between API URLs and Clients + fallback on OAI for generic IP addresses)